docs/user-guide/glossary.md

```diff
@@ -158,6 +158,20 @@ of type `int`.
 
 For example, a battery with a component ID of **5**.
 
+### Component Graph
+
+A [graph](https://en.wikipedia.org/wiki/Graph_(abstract_data_type))
+representation of the configuration in which the electrical components in a
+microgrid are connected with each other.  Some of the ways in which the SDK uses
+the component graph are:
```

Comment:
shouldn't be component**s** graph instead? (Same for the title if this comment applies)

Reply:
No,  it is like a guest list, or a paper clip